tests/test_helper.js

```diff
@@ -105,9 +105,7 @@ function assertValidHTML(response, done) {
                 return false;
             })
             .filter((e) => {
-                const ignores = [
-                    /^Error: Attribute.+color.+not allowed on element.+link.+at this point./
-                ];
+                const ignores = [];
```

Comment:
@jmervine: this is irrelevant to this PR, I just pushed it here so that I don't have to rebase. I can do it in develop and rebase if you prefer.